common/utils/util.js

```diff
@@ -88,6 +88,22 @@ const catchAndReThrowError = (msg, error) => {
   throw newError
 }
 
+const isValidDate = (day, month, year) => {
```

Comment:
year, month, day would make more sense. I'm going to raise a ticket to stick some defaults in too so we can utilise this more 👍

Reply:
Created to re-use this function:
https://dsdmoj.atlassian.net/browse/SP-578